polars/polars-time/src/chunkedarray/utf8/mod.rs

```diff
@@ -482,7 +482,6 @@ pub trait Utf8Methods: AsUtf8 {
                         // Safety:
                         // fmt_len is correct, it was computed with this `fmt` str.
                         unsafe { self::strptime::parse(s.as_bytes(), fmt.as_bytes(), fmt_len) }
-                            .or_else(|| NaiveDateTime::parse_from_str(s, &fmt).ok())
```

Comment:
anything which `fmt_len` returns `Some` for should be parseable by `strptime::parse`, so it should be OK to remove this fallback